ts/components/AbortOnboardingModal.tsx

```diff
@@ -22,7 +23,7 @@ class AbortOnboardingModal extends React.PureComponent<Props> {
       <Modal visible={true} onRequestClose={this.props.onClose}>
         <Container>
           <AppHeader noLeft={true}>
-            <Right>
+            <Right style={styles.rightOnly}>
```

Comment:
The name "rightOnly" suggest this style is used when only the "Right" component is rendered but from the code is not the case.